### ISSUE TITLE
[BUG] hamlet engine env failing on tram version 27 June

### ISSUE THREAD

**Issue by ken9ross** [hamlet-io/executor-python#177]
## Current Behaviour
```
$ hamlet engine env
# run eval $(hamlet engine env) to set variables
export GENERATION_ENGINE_DIR=""
export GENERATION_PLUGIN_DIRS=""
export GENERATION_BIN_DIR=""
export GENERATION_BASE_DIR=""
export GENERATION_DIR=""
export AUTOMATION_DIR=""
export AUTOMATION_BASE_DIR=""
```
Also environment variables AUTOMATION_* and GENERATION_DIR refer to locations that do not exist

## Expected Behaviour
Expected valid values in both base docker container and hamlet engine env

## Steps to Reproduce

1. get hamletio/hamlet:latest
2. hamlet engine env and/or ls -ltr $GENERATION_DIR
3. hamlet engine install-engine tram
4. hamlet engine env and/or ls -ltr $GENERATION_DIR

**Comment by roleyfoley** [comment]
This should have been fixed in #176 which makes sure the global engine is always installed. Just checking a build of the docker container to see if that has been resolved

**Comment by roleyfoley** [comment]
Created a local build of the docker-hamlet container and confirmed that the issue had been resolved. Created a new build of the container and using that image everything was fine. 

It appears that the docker-hamlet build wasn't up to date with the latest version of the cli. 

Will call this closed as the issue is now resolved
